### DISCUSSION TITLE
Microphone volume low?

### DISCUSSION THREAD

**Discussion by chaseadam** [juha-h/baresip-studio#386] [Q&A] [answered]
The app experience for me has been great, but my callers report very low volume from me while on a call. If I cup my hand around the bottom of the phone to increase my speaking volume to the microphone, they report acceptable volume.

Is there an adjustment for the microphone sensitivity?

Thanks!

Environment:
Essential PH-1 with DivestOS (Android 13)

**Comment by juha-h** [comment, marked as answer]
I have not found any way to programmatically adjust microphone gain.  It is only possible to adjust speaker volume.  If the other party of your call is using baresip, there exists Default Call Volume audio setting.  Speaker volume can also be changed by adjusting Media Volume of the device.
